protobuff/archive.proto

```diff
@@ -116,9 +121,14 @@ message SkippedTicksIntervalList {
   repeated SkippedTicksInterval skipped_ticks = 1;
 }
 
-message QuorumTickData {
+message QuorumTickDataV2 {
```

Comment:
why do we have v1 and v2? this version of archive should be used with a clean(migrated) version of storage so we shouldn't keep any reference to the old quorum format, or am I missing something?